sdcm/sla/sla_tests.py

```diff
@@ -123,12 +120,29 @@ def validate_role_service_level_attributes_against_db():
                 role_for_attach.attach_service_level(new_service_level)
                 role_for_attach.validate_role_service_level_attributes_against_db()
 
-                # Workaround for issue https://github.com/scylladb/scylla-enterprise/issues/2572
-                if restart_scylla:
-                    nodes_for_restart = [node for node in tester.db_cluster.nodes if node.jmx_up() and node.db_up()
-                                         and not node.running_nemesis]
-                    tester.db_cluster.restart_binary_protocol(nodes=nodes_for_restart)
-                # End workaround
+                for node in tester.db_cluster.nodes:
```

Comment:
Do you plan to handle this issue (https://github.com/scylladb/scylla-enterprise/issues/2572)? If yes - we still need the comment that it is workaround. So when the issue will be fixed - we will remove it from the code

Reply:
added a comment